package ex01;

public class AppEx04 {
    public static void main(String[] args) {
        throw new RuntimeException("강제 오류발생");
    }
}
